src/families/ripple/bridge/js.js

```diff
@@ -183,10 +183,16 @@ const broadcast = async ({ signedOperation: { signature, operation } }) => {
   const api = apiForEndpointConfig(RippleAPI);
   try {
     await api.connect();
-    const submittedPayment = await api.submit(signature);
+    const submittedPayment = await api.request("submit", {
+      tx_blob: signature,
+      fail_hard: false,
+    });
 
-    if (submittedPayment.resultCode !== "tesSUCCESS") {
-      throw new Error(submittedPayment.resultMessage);
+    if (
+      submittedPayment.engine_result !== "tesSUCCESS" &&
+      submittedPayment.queued !== true
+    ) {
+      throw new Error(submittedPayment.engine_result_message);
```

Comment:
Maybe not for now, but we could potentially map the errors here for more granularity 🤔 
cc @gre